bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -204,7 +204,7 @@ case object SemesterConverter2023BTo2024A extends SemesterConverter {
 
   def transformNifsName(name: String) = name match {
     case nifsNameRegexAo(a, b, _) => s"""GNIRS Spectroscopy $a$b 0.15"/pix 111 l/mm grating None LR-IFU &lt; 2.5um"""
-    case nifsNameRegexNonAo(_)    => s"""GNIRS Spectroscopy  0.15"/pix 111 l/mm grating None LR-IFU &lt; 2.5um"""
+    case nifsNameRegexNonAo(_)    => s"""GNIRS Spectroscopy LGS/PWFS1 0.15"/pix 111 l/mm grating None LR-IFU &lt; 2.5um"""
```

Comment:
It's `NonAo` but uses LGS?

Reply:
This has been clarified, we'll change this logic